src/Doctrine/TranslatableType.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Webfactory\Bundle\PolyglotBundle\Doctrine;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\Type;
+use RuntimeException;
+
+/**
+ * Doctrine type to support mapping database string types (VARCHAR, TEXT etc.)
+ * as the "primary" translation values into fields that declare {@see TranslatableInterface}
+ * as their only type.
+ */
+class TranslatableType extends Type
+{
+    public const TYPE = 'translatable';
```

Comment:
Maybe it should be called `translatable_string` instead?

That would emphasise the similarities with Doctrine's own `string` type (which is for example that by default, it will use the same database definition), hint that you have to use `Translatable<string>` as a type for the field, and avoid confusions in case someone creates Translatable Doctrine types for other types than strings (e. g. `translatable_json` or `translatable_download`).

Reply:
I like that, could you please change it?

Reply:
Updated.